operators/Makefile

```diff
@@ -28,9 +28,11 @@ LATEST_RELEASED_IMG ?= "docker.elastic.co/eck/$(NAME):0.8.0"
 ifneq ($(findstring gke_,$(KUBECTL_CLUSTER)),)
 	REGISTRY ?= eu.gcr.io
 	REPOSITORY = ${GCLOUD_PROJECT}
-endif
-# default to local registry
-ifeq ($(REGISTRY),)
+else ifneq ($(findstring azmk8s.io:443,$(shell kubectl config view --minify -o=jsonpath={.clusters[*].cluster.server} 2> /dev/null)),)
```

Comment:
I would prefer to expose `REGISTRY` and `REPOSITORY` in Jenkins pipeline (Jenkinsfile-aks) and set them explicitly

Reply:
But it's just a personal preference of mine

Reply:
Some concerns:
1. registry is a direct derivative of the provider that is specified in the Jenkinsfile already,
1. if we run with multiple providers via same Jenkinsfile we need to distinguish between registries for each, 
1. we might argue the same for every setting.

How about revisiting when we will be moving to common docker repo?